app/api/submission/Representative.scala

```diff
@@ -26,8 +26,10 @@ object Representative {
       r =>
         RepresentativeType05(
           r.eori.value,
-          r.capacity.code,
-          Some(ContactPersonType05(r.name, r.phone, None))
+          "2",
```

Comment:
Will this always be 2?

Reply:
I believe so from looking at ticket CTCP-2329